integrations/cassandra-integration.mdx

```diff
@@ -3,7 +3,9 @@ title: Cassandra
 description: Cassandra Integration - Setup & Installation Docs | Middleware
 ---
 
-The Cassandra Database Integration provides seamless access to performance metrics and enables users to effortlessly visualize crucial insights in a single dashboard. The integration works by pulling metrics from the database through the Middleware Host Agent.
+The Cassandra Database Integration provides seamless access to your Cassandra Database performance metrics and enables users to effortlessly visualize crucial insights in a single dashboard. This integration is useful for teams that want to visualize total compaction tasks over time, read/write latency, storage resource availability, and much more.
```

Comment:
Only need to use "Cassandra" once. Can say "The Cassandra Database Integration provides seamless access to your database" also notice how without Cassandra in there I lowercased "database"